qa/L0_backend_config/test.sh

```diff
@@ -0,0 +1,146 @@
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+SERVER=/opt/tritonserver/bin/tritonserver
+SERVER_TIMEOUT=20
+source ../common/util.sh
+
+
+SERVER_LOG_BASE="./inference_server"
+rm -f $SERVER_LOG_BASE*
+
+NEGATIVE_PARSE_ARGS=("--model-repository=`pwd`/models --backend-config=,default-max-batch-size=3 --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=default-max-batch-size= --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=default-max-batch-size --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=tensorflow,default-max-batch-size= --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=tensorflow,default-max-batch-size --strict-model-config=false --log-verbose=1" \
+)
+
+POSITIVE_DEFAULT_ARGS="--model-repository=`pwd`/models --strict-model-config=false --log-verbose=1" 
+
+POSITIVE_TEST_ARGS=("--model-repository=`pwd`/models --backend-config=tensorflow,default-max-batch-size=5 --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=default-max-batch-size=6 --strict-model-config=false --log-verbose=1" \
+                    "--model-repository=`pwd`/models --backend-config=default-max-batch-size=7 --backend-config=tensorflow,default-max-batch-size=8 --strict-model-config=false --log-verbose=1" \
+                    
+                    
+)
+
+# These integers correspond to the expected default-max-batch-size which gets set 
+# in the POSITIVE_TEST_ARGS
+POSITIVE_TEST_ANSWERS=("default-max-batch-size,5" "default-max-batch-size,6" "default-max-batch-size,8")
```

Comment:
You can remove "default-max-batch-size" from here since it is common

Reply:
Took out the common "default-max-batch-size" and updated the `awk` to parse the line to match